test/integration/instance_with_attached_disk/controls/docker.rb

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+require "docker"
```

Comment:
The `docker` method is provided by InSpec. What is the purpose of this import?

Reply:
Fixed